www/src/content/blog/2024-12-12-honc-uptime-monitor.mdx

```diff
@@ -0,0 +1,239 @@
+---
+title: "Building an uptime monitor using HONC and Cloudflare Workers"
+description: An uptime monitor built with Hono, Durable Objects,D1, and Cloudflare Workers
+slug: honc-up-time-monitor
+date: 2024-12-12
+author: Micha "mies" Hernandez van Leuffen
+tags:
+  - HONC
+  - uptimemonitor
+  - Honojs
+---
+
+import { LinkCard } from "@astrojs/starlight/components";
+
+In this tutorial, we'll build a serverless website uptime monitoring application using Cloudflare Workers. Our application will periodically check websites' availability, track response times, and store the results in a serverless database. We'll use the HONC stack: Hono (a lightweight web framework), OpenTelemetry (for observability), and Cloudflare's suite of serverless products.
```

Comment:
not sure about OpenTelemetry? Should it not mention the components of the HONC stack?

Reply:
I can't see Otel as part of the application code? Or should this relate to fiberplane's hono client that uses Otel?